### PR TITLE
win_dns_record PTR docs update

### LINKED ISSUE #60068: win_dns_record incorrectly creates PTR records
##### SUMMARY

When specifying a PTR record, the module incorrectly creates the entry.

  - name: Create PTR Record
    win_dns_record:
      computer_name: "msdnsserver.example.com"
      name: "10.202.64.62"
      state: present
      type: "PTR"
      value: "awx_test.example.com"
      zone: "10.in-addr.arpa"    

Entry created:

10.62.64.202.10     Pointer (PTR)     awx_test.example.com

To correctly create the record I used the following format:

  - name: Create PTR Record
    win_dns_record:
      computer_name: "msdnsserver.example.com"
      name: "62.64.202"
      state: present
      type: "PTR"
      value: "awx_test.example.com"
      zone: "10.in-addr.arpa"    

which created record:

10.202.64.62     Pointer (PTR)     awx_test.example.com

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

win_dns_record
##### ANSIBLE VERSION

```paste below
# ansible --version
ansible 2.8.3
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /bin/ansible
  python version = 2.7.5 (default, Jun 11 2019, 12:19:05) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]

```

##### CONFIGURATION

```paste below
# ansible-config dump --only-changed
ANSIBLE_PIPELINING(/etc/ansible/ansible.cfg) = True
DEFAULT_LOAD_CALLBACK_PLUGINS(/etc/ansible/ansible.cfg) = True
DEFAULT_LOG_PATH(/etc/ansible/ansible.cfg) = /var/log/ansible.log
SHOW_CUSTOM_STATS(/etc/ansible/ansible.cfg) = True

```

##### OS / ENVIRONMENT

Red Hat Enterprise Linux Server release 7.6 (Maipo)

##### STEPS TO REPRODUCE

```yaml
  - name: Create PTR Record
    win_dns_record:
      computer_name: "msdnsserver.example.com"
      name: "10.202.64.62"
      state: present
      type: "PTR"
      value: "awx_test.example.com"
      zone: "10.in-addr.arpa"    
```

##### EXPECTED RESULTS

10.202.64.62     Pointer (PTR)     awx_test.example.com

##### ACTUAL RESULTS

```paste below
10.62.64.202.10     Pointer (PTR)     awx_test.example.com
```

### PR DESCRIPTION
**Pull request by ShachafGoldstein**
##### SUMMARY
Fixes: #60068

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
win_dns_record

### CODE REVIEW

**Comment by ShachafGoldstein** [comment]
CI doesn't work right now, ill try again later

File: lib/ansible/modules/windows/win_dns_record.py [hunk 1]
```diff
@@ -76,6 +77,13 @@
     value: "cgyl1404p.amer.example.com"
     zone: "amer.example.com"
 
+- name: PTR example
+  win_dns_record:
+    name: "db1"
```
Comment (angystardust) [inline lib/ansible/modules/windows/win_dns_record.py]: @ShachafGoldstein I think that you inverted the `name` and the `value` parameters here:
It should be:
```
name: "1.1.1"
```

File: lib/ansible/modules/windows/win_dns_record.py [hunk 2]
```diff
@@ -76,6 +77,13 @@
...
+    type: "PTR"
+    value: "1.1.1"
```
Comment (angystardust) [inline lib/ansible/modules/windows/win_dns_record.py]: Same here. It should be:
`value: "db1"`

**Comment by ShachafGoldstein** [comment]
Correct thanks

**Comment by acozine** [comment]
@angystardust @ShachafGoldstein are you sure the `name` and `value` values are correct in the (changed) new example? If so, the example above it is incorrect:

```
- name: Create database server alias
  win_dns_record:
    name: "db1"
    type: "CNAME"
    value: "cgyl1404p.amer.example.com"
    zone: "amer.example.com"
```

I suspect the original formulation of `name: "db1"` and `value: "1.1.1"` is correct, though I have no way to test my suspicions.

**Comment by ShachafGoldstein** [comment]
For dns ptr records the value is the name and the name is the ip

In all other dns record types the name is the name and value is ip

**Comment by acozine** [comment]
Thanks for clarifying how the code works @ShachafGoldstein. I'll merge this, since the documentation reflects the current state of the code.

**Pull request merged**